pages/guides/practical-guides/flexberry-ember/gpg_setting-language-and-structure.ru.md

```diff
@@ -0,0 +1,226 @@
+
+**По итогу** данного шага у вас должна получиться следующая диаграмма:
+
+![Диаграмма с настроенными именами классов](/images/pages/guides/flexberry-ember/1-setting-language-and-structure/4.png)
+
+Для того, чтобы в интерфейсе приложения, в отличии от кода, фигурировали русские наименования, требуется задать свойство Caption у класса. Для того, чтобы это сделать, выполните:
+
+*`Order → [ПКМ] → ORM: Редактировать свойства`*
+
+![Настройка заголовка класса](/images/pages/guides/flexberry-ember/1-setting-language-and-structure/5.png)
+
+Здесь мы можем видеть, что верное именование уже проставлено. Причиной этому служит тот факт, что создавали мы классы именно на русском языке и при создании автоматически были проставлены значения свойств “Description” И “Caption”.
+
+Если вы будете создавать классы изначально на английском, не забывайте уточнять отображение их заголовков.
+
+## Именование атрибутов
+
+Рассмотрим переименование атрибутов на примере класса Order (Заказ). Для этого выполним следующие команды:
+
+*`Order → [ПКМ] → ORM: Редактировать свойства → Атрибуты`*
+
+В открывшемся окне изменим поля следующим образом:
+
+![Вкладка редакирования атрибутов](/images/pages/guides/flexberry-ember/1-setting-language-and-structure/6.png)
+
+> Статус → Status  
+> ДатаОтгрузки → ShipmentDate  
+> ДатаОплаты → PaymentDate  
+> Цена → TotalSum
+
+{% include important.html content="Так как мы изменили наименование класса спискового типа “СтатусЗаказа” на “OrderStatus”, мы также должны изменить тип свойства для статуса заказа. То же нужно будет сделать для Invoice (Накладная).  
+Атрибуты класса можно редактировать непосредственно из диаграммы." %}
+
+---
+**_Самостоятельно_** переименуйте атрибуты остальных классов следующим образом:
+
+**Order**
+
+> Отменить() → Cancel() *
+
+**OrderItem**
+> Количество → Amount  
+> ЦенаСНалогами → PriceWTaxes  
+> Сумма → TotalSum
+
+**Invoice**
+> Статус → Status  
+> ДатаВремяОтгрузки → ShipmentDateTime  
+> Сумма → TotalSum  
+> Вес → TotalWeight  
+> Примечание → Note  
+> ФИОПолучателя → CustomerName  
+> **Особо**: СостояниеНакладной → InvoiceStatus
+
+**InvoiceItem**
+> Количество → Amount  
+> Вес → Weight  
+> Цена → Price  
+> Сумма → TotalSum
+
+**Storehouse**
+> Номер → Number  
+> Адрес → Address
+
+**StoreProduct**
+> Количество → Amount
+
+**Product**
+> КодТовара → ProductCode  
+> Название → Name  
+> ЕдиницаИзмерения → Measure  
+> Описание → Description  
+> Цена → Price  
+
+**Employee**
+> ТабельныйНомер → Number  
+> Фамилия → LastName  
+> Имя → FirstName  
+> Отчество → MiddleName  
+> Телефон → PhoneNumber  
+> EMail → Email
+
+**Document**
+> Номер → Number  
+> ДатаЗаполнения → CreateDate
+---------------------------------------------------
```

Comment:
Need an additional empty line here.
Otherwise it will look like [this](https://yadi.sk/d/xLPDNdCktwxMNQ)

Reply:
done